Std/Data/String/Lemmas.lean

```diff
@@ -60,6 +60,9 @@ instance : Std.BEqOrd String := .compareOfLessAndEq String.lt_irrefl
 
 @[simp] theorem data_append (s t : String) : (s ++ t).1 = s.1 ++ t.1 := rfl
 
+@[simp] theorem cons_append (c : Char) (cs : List Char) (t : String) :
```

Comment:
I'm skeptical this should be a `@[simp]` lemma.

Reply:
You don't seem to have removed this (or explained why you think it should stay).